docs/instrumentations/java/java.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: "Java Native Instrumentation"
```

Comment:
I think perhaps we can add a short paragraph to explain it, so users can easily figure out which java documentation is relevant to their setup